model/model.js

```diff
@@ -96,4 +96,27 @@ exports.insertCommentByArticleId = (articleId, newComment) => {
     .catch((err) => {
       next(err)
     })
+}
+
+exports.updateArticleById = (articleId, inc_votes) => {
+  const articleIdNum = Number(articleId)
+  const regex = new RegExp(/[^\d]/g)
+  if (regex.test(articleIdNum)) {
+    return Promise.reject({status: 400, msg: 'Bad request'})
+  }
```

Comment:
This would be taken care of by SQL so you could just pass the number - without using Number() - and then handle the PSQL error that you would receive back.